packages/webapp/pages/onboarding.tsx

```diff
@@ -325,7 +325,7 @@ export function OnboardPage(): ReactElement {
       OnboardingStep.ContentTypes,
       OnboardingStep.ReadingReminder,
     ].includes(activeScreen);
-    if (isLastStepBeforePlus && !isIOSNative() && isValidRegion) {
+    if (isLastStepBeforePlus && !isIOSNative() && isValidRegion && !isFunnel) {
```

Comment:
If user is from funnel should be already plus. Why not checking for it? @idoshamun

Reply:
that's a good point. it's a better alternative actually